handlers/handlers.go

```diff
@@ -34,17 +35,50 @@ func GeographyStart(cfg config.Config, rc RenderClient) http.HandlerFunc {
 // GetArea Handler
 func GetArea(ctx context.Context, cfg config.Config, c Clients) http.HandlerFunc {
 	return dphandlers.ControllerHandler(func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-		func(w http.ResponseWriter, req *http.Request, lang, collectionID, accessToken string) {
-			vars := mux.Vars(req)
-			areaID := vars["id"]
-			//acceptedLang := req.Header.Get("Accepted-Language")
-			areaData, err := c.AreaApi.GetArea(ctx, accessToken, "", collectionID, areaID)
-			if err != nil {
-				log.Error(ctx, "Fetching Area Data", err)
-			}
-			basePage := c.Render.NewBasePageModel()
-			model := mapper.CreateAreaPage(basePage, areaData)
-			c.Render.BuildPage(w, model, "area-summary")
-		}(w, req, lang, collectionID, accessToken)
+		GetAreaViewHandler(w, req, ctx, c, lang, collectionID, accessToken)
 	})
 }
+
+func GetAreaViewHandler(w http.ResponseWriter, req *http.Request, ctx context.Context, c Clients, lang, collectionID, accessToken string) {
+	var err error
+	var relationsErr error
+	var areaData areas.AreaDetails
+	var relationsData []areas.Relation
+	vars := mux.Vars(req)
+	areaID := vars["id"]
+	acceptedLang := req.Header.Get("Accept-Language")
+	var wg sync.WaitGroup
+	wg.Add(2)
+	// Remote requests
+	go func() {
+		defer wg.Done()
+		areaData, err = c.AreaApi.GetArea(ctx, accessToken, "", collectionID, areaID, acceptedLang)
+		if err != nil {
+			log.Error(ctx, "Fetching Area Data", err)
+			return
+		}
+	}()
+	go func() {
+		defer wg.Done()
+		// Create a new local error variable otherwise we will incur a race condition when other goroutines access it
+
+		relationsData, relationsErr = c.AreaApi.GetRelations(ctx, accessToken, "", collectionID, areaID, acceptedLang)
+		if relationsErr != nil {
+			log.Error(ctx, "Fetching area relations data", relationsErr)
+			return
+		}
+	}()
+
+	wg.Wait()
+	if err != nil || relationsErr != nil {
```

Comment:
I have added a helper function for this in another pr
https://github.com/ONSdigital/dp-frontend-area-profiles/blob/7261ac04cd571d4c3cfe8a21930e2738e2108540/utils/requests.go